vault/expiration.go

```diff
@@ -640,7 +640,7 @@ func (m *ExpirationManager) RevokeByToken(te *logical.TokenEntry) error {
 
 	// te.Path should never be empty, but we check just in case
 	if te.Path != "" {
-		saltedID, err := m.tokenStore.SaltID(m.quitContext, te.ID)
+		saltedID, err := m.tokenStore.SaltID(ctx, te.ID)
```

Comment:
I'm not sure about this change. This may well take more than 90 seconds if there are a lot of external leases and ideally we'd let it go until forced to stop, which suggests keeping m.quitContext in this function.

Reply:
This is another example where the caller of `RevokeByToken` will pass in the appropriate context.